src/urdf_parser_py/urdf.py

```diff
@@ -370,6 +374,8 @@ def __set_visual(self, visual):
             self.visuals[0] = visual
         else:
             self.visuals.append(visual)
+        if visual:
+            self.add_aggregate('visual', visual)
```

Comment:
Yes, it makes sense to me. I add `add_visual` or `add_collision` methods and refactored the unit tests.

Do we also need `remove_visual(visual)` or `remove_collision(collision)` methods?

Reply:
> Do we also need `remove_visual(visual)` or `remove_collision(collision)` methods?

Yeah, I think that would be a good idea.  Thanks!

Reply:
Actually, it was a good thing to add these methods, since in the previous version I added the objects twice...

It is actually sufficient to simply call `add_aggregate` or `remove_aggregate`. There is no need to manipulate the lists `Link.collisions` or  `Link.visuals` directly.